Mathlib/Combinatorics/SimpleGraph/Prod.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: George Peter Banyard, Yaël Dillies, Kyle Miller
 -/
 import Mathlib.Combinatorics.SimpleGraph.Path
+import Mathlib.Combinatorics.SimpleGraph.Metric
```

Comment:
I'd be tempted to make this import in the other direction, but perhaps @kmill has stronger opinions

Reply:
I think this local straightening of the import graph is bad, but easier to work against once we have enough material in a given folder to decide what the "primitives" of that folder are, and what's derived. Therefore I am happy to leave this as is for now